src/ProjectorMonteCarloProblem.jl

```diff
@@ -0,0 +1,261 @@
+"""
+    FCIQMC()
+Algorithm for the full configuration interaction quantum Monte Carlo (FCIQMC) method.
+To be passed as the `algorithm` keyword argument to [`ProjectorMonteCarloProblem`](@ref).
+"""
+struct FCIQMC end
```

Comment:
Should `shift_strategy` and `timestep_strategy` (and others?) be stored in here? Other algorithms may not use a shift or timestep.